README.md

```diff
@@ -124,7 +124,7 @@ Open the `Vireo_Xcode/VireoEggShell.xcodeproj`
 
 # Running Tests
 ## Running VTR Tests
-The VTR test suite runs .via files located in the `test-it/ViaTests` directory, captures the standard output during the execution, and compares it to the .vtr output file located in `test-it/ExpectedResults`. The VTR test suite is the primary test suite for testing execution behavior for Vireo.
+The VTR (Vireo test runner) test suite runs .via files located in the `test-it/ViaTests` directory, captures the standard output during the execution, and compares it to the .vtr output file located in `test-it/ExpectedResults`. The VTR test suite is the primary test suite for testing execution behavior for Vireo.
```

Comment:
vtr stands for vireo test result, ie the vtr files on disk.

The VTR test suite is the set of tests that have paired vtr and via files. This is in contrast to the karma test suite.

"The Vireo Test Runner test suite" sounds like a test suite for the vireo test runner.